src/helpers/breakpoints.js

```diff
@@ -0,0 +1,8 @@
+const BREAKPOINTS = [
+  {
+    name: 'large',
+    media: '(min-width: 900px)',
+  },
+];
+
+export default BREAKPOINTS;
```

Comment:
Why you didn't add all breakpoints?
Please add them all. 
First define names like this
```
const name = {
  SMALL: 'small',
  MEDIUM: 'medium',
  LARGE: 'large',
};
```
then values
```
const value = {
  [name.LARGE]: ???,
  [name.MEDIUM]: ???,
  [name.SMALL]: ???,
};
```
and then define all of them like you did but using those predefined names / values.